sklearn/utils/estimator_checks.py

```diff
@@ -640,7 +641,7 @@ def check_fit1d_1feature(name, estimator_orig):
 
 @ignore_warnings
 def check_fit1d_1sample(name, estimator_orig):
-    # check fitting 1d array with 1 feature
+    # check fitting 1d array with 1 sample
```

Comment:
Indeed. I will replace the two fit_1d checks, `check_fit1d_1sample` and `check_fit1d_1feature`, with only one `check_fit1d` testing that a `ValueError` is raised.